### PR TITLE
[author] [new] Add vidom by single package.json

### PR DESCRIPTION
**Pull request by dfilatov**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/dfilatov/vidom
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/vidom
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 157
   - Count of watchers: 15
   - Count of forks: 9
 * NPM download stats (optional):
   - Downloads in the last day: 
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vidom/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vidom",
+  "filename": "vidom.min.js",
+  "version": "0.3.3",
```
Comment (dfilatov) [inline ajax/libs/vidom/package.json]: Fixed.

**Comment by dfilatov** [comment]
@PeterDaveHello Thanks!

Could you explain, will the current last version be automatically get from npm and uploaded to cdn? Or should I do it manually the first time?

**Comment by PeterDaveHello** [comment]
@dfilatov no probs. BTW since we are working on some issues on the CDN today, so your lib will not be available on the CDN immediately, sorry for the inconvience caused.

**Comment by dfilatov** [comment]
@PeterDaveHello Vidom is already available via cdn!
Thank you so much for cdnjs!

**Comment by PeterDaveHello** [comment]
@dfilatov No probs, would you like to add the info in readme or website? So that more users can know that.

**Comment by dfilatov** [comment]
> would you like to add the info in readme or website?
done

**Comment by PeterDaveHello** [comment]
@dfilatov Thanks

**Pull request merged**